xbmc/video/VideoDatabase.cpp

```diff
@@ -2593,6 +2593,28 @@ bool CVideoDatabase::GetSetInfo(int idSet, CVideoInfoTag& details, CFileItem* it
   return false;
 }
 
+bool CVideoDatabase::GetSetInfo(const std::string& title, CSetInfoTag& tag) const
```

Comment:
There already is a GetSetInfo function which handles very different information. Maybe GetSetIntoTag would be better